utils/detect_sound.py

```
@@ -0,0 +1,55 @@
+import sys
+import os
+import subprocess
+import glob
+
+def get_process_names():
+    """Return a set of running process names."""
+    try:
+        # pgrep -a . returns full command lines
+        output = subprocess.check_output(["pgrep", "-a", "."], text=True).splitlines()
```

Ruff (0.14.10)

Starting a process with a partial executable path

(S607)